example/android/app/src/main/AndroidManifest.xml

```diff
@@ -46,10 +46,14 @@
             android:value="2" />
         <meta-data
             android:name="CLEVERTAP_ACCOUNT_ID"
-            android:value="TEST-65R-487-R85Z"/>
+            android:value="8RK-KR9-976Z"/>
```

Comment:
@shivamsharma2710 remove credentials before merging

Reply:
This is done.